### PR TITLE
Adds a button on the UI for the Web-Map

### PR DESCRIPTION
**Pull request by Joseph-Case**
Throws a button for the web-map right in peoples faces because ~~I want my work seen~~ this is a useful tool. Ignore the 7 commits it took for me to make this.

:cl: AffectedArc07
add: The UI now has a dedicated button for the webmap.
/:cl:

I feel this would be a good addition as its a useful tool for learning the map

### CODE REVIEW

**Comment by SamHPurp** [comment]
Image of it please?

File: interface/skin.dmf [hunk 1]
```diff
@@ -929,26 +949,16 @@ window "infowindow"
...
 		pos = 96,5
-		size = 88x20
+		size = 86x20
 		anchor1 = 15,0
 		anchor2 = 29,0
 		background-color = none
 		saved-params = "is-checked"
 		text = "Rules"
 		command = "rules"
-	elem "wiki"
```
Comment (FlattestGuitar) [inline interface/skin.dmf]: You removed the wiki button. Don't remove the wiki button.

**Comment by Joseph-Case** [comment]
I will get one when I am home

**Comment by Joseph-Case** [comment]
image

**Comment by FlattestGuitar** [comment]
Too small on my 16:10. Make it bigger.

**Comment by Joseph-Case** [comment]
Can I just change the text to "Map" and leave it at that?

**Comment by Joseph-Case** [comment]
I will attempt to make them all the same size at some point

**Comment by Birdtalon** [comment]
The only reason I don't like this is because it links to somewhere outside of the sphere of control and therefore we cannot guarantee that it's necessarily a trusted site. The web-map would have to be brought in-house to our own infrastructure for me to be comfortable with it.

**Comment by Joseph-Case** [comment]
Ok, The code is available on the github fully working after dinking around with it today, I will close this for now, when you want me to add the code to its own web github, just ask and I will PR it over.

**Pull request closed without merging**